definitions/pipelines/immuno.cwl

```diff
@@ -59,32 +59,103 @@ inputs:
             - string
             - File
         secondaryFiles: [.fai, ^.dict, .amb, .ann, .bwt, .pac, .sa]
+        label: "reference: Reference fasta file for a desired assembly"
+        doc: |
+          reference contains the nucleotide sequence for a given assembly (hg37, hg38, etc.)
+          in fasta format for the entire genome. This is what reads will be aligned to.
+          Appropriate files can be found on ensembl at https://ensembl.org/info/data/ftp/index.html
+          When providing the reference secondary files corresponding to reference indices must be
+          located in the same directory as the reference itself. These files can be created with
+          samtools index, bwa index, and picard CreateSequenceDictionary.
 
     tumor_sequence:
         type: ../types/sequence_data.yml#sequence_data[]
+        label: "tumor_sequence: yml file specifying the location of MT sequencing data"
+        doc: |
+          tumor_sequence is a yml file for which to pass information regarding
```

Comment:
I don't think it's quite right to say that this is a `yml file`.  It's a data structure that happens to be described in `sequence_data.yml`, but the data structure itself needn't be provided in its own file.  It's part of the regular inputs.

Reply:
I might suggest also modifying (or working with @zlskidmore or @bryanfisk) the somatic_exome.cwl doc/label for these same inputs:
https://github.com/genome/analysis-workflows/blob/cfdcd0735e8fb1ac0b97f1a5cbe0cc7e00033dd0/definitions/pipelines/somatic_exome.cwl#L39